pkgs/applications/networking/cluster/tektoncd-cli/default.nix

```diff
@@ -24,6 +24,8 @@ buildGoModule rec {
   excludedPackages = "\\(third_party\\|cmd/docs\\)";
 
   preCheck = ''
+    # Some tests try to write to the home dir
+    export HOME="$TMPDIR"
```

Comment:
Hum, we should report this upstream, because we really shouldn't 😓

Reply:
Yeah can do. Ill get a full error log with the readonly home dir and post an issue